src/components/layout/MainMenu/RightMenu.jsx

```diff
@@ -36,41 +51,133 @@ const RightMenu = () => {
   const userIsDelegator = currentUser.isDelegator || !delegateWhitelistEnabled;
   const userIsCampaignManager = currentUser.isProjectOwner || !projectOwnersWhitelistEnabled;
   const userIsReviewer = currentUser.isReviewer || !reviewerWhitelistEnabled;
+  const userAddress = currentUser.address;
 
   const { pathname } = useLocation();
 
+  const getUserTraces = () => {
+    TraceService.getUserTraces({
+      traceStatus: Trace.ACTIVE,
+      ownerAddress: userAddress,
+      recipientAddress: userAddress,
+      skipPages: 0,
+      itemsPerPage: 0,
+      onResult: resp => {
+        setUserTotalTraces(resp.total);
+      },
+      onError: err => {
+        ErrorPopup('Something went wrong on getting user Traces!', err);
+      },
+    }).then();
+  };
+
+  const getUserDonations = () => {
+    GetDonationsService.getUserDonations({
+      currentUser,
+      itemsPerPage: 0,
+      skipPages: 0,
+      subscribe: false,
+      onResult: resp => {
+        setUserTotalDonations(resp.total);
+      },
+      onError: err => {
+        ErrorPopup('Something went wrong on getting user donations!', err);
+      },
+    });
+  };
+
+  const getUserDelegations = () => {
+    LoadProjectsInfo({ userAddress })
```

Comment:
@RamRamez I know its legacy code, but using `LoadProjectsInfo` is not efficient. It loads traces too which are not needed